packages/spaces/src/SpacesLink.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import nativeForm from '@availity/native-form';
+import { Card, CardLink, CardBody, Media, CardText, CardTitle, Badge } from 'reactstrap';
+import dayjs from 'dayjs';
+import AppIcon from '@availity/app-icon';
+import Icon from '@availity/icon';
+import { useSpaces, useSpacesContext } from './Spaces';
+import { updateUrl, updateTopApps } from './helpers';
+import { useModal } from './modals/ModalProvider';
+
+export const useLink = spaceId => {
+  const { clientId } = useSpacesContext();
+  const openModal = useModal();
+
+  const [{ id, type, name, description, parents = [], metadata = {}, link, ...space } = {}] = useSpaces(spaceId);
+
+  const parentPayerSpaces = parents.filter(p => p.type === 'space');
+
+  // TODO Abstract this
+  const legacySso = () => {
+    openModal('DISCLAIMER_MODAL', {
+      disclaimerId: metadata.disclaimerId,
+      name,
+      spaceType: type,
+      id,
+      title: name,
+      description,
+      link,
+    });
+  };
+
+  const linkSso = event => {
+    if (metadata && metadata.ssoId) {
+      event.preventDefault();
+      const options = link.target ? { target: link.target } : undefined;
+
+      const attributes = {
+        X_Client_ID: clientId,
+        X_XSRF_TOKEN: document.cookie.replace(/(?:(?:^|.*;\s*)XSRF-TOKEN\s*=\s*([^;]*).*$)|^.*$/, '$1'),
+        spaceId,
+      };
+
+      updateTopApps(id, type);
+
+      nativeForm(metadata.ssoId, attributes, options);
+      return false;
+    }
+    return false;
+  };
+
+  const mediaProps = {};
+
+  if (metadata.ssoId) {
+    mediaProps.onClick = linkSso;
+  } else if (metadata.disclaimerId) {
+    mediaProps.onClick = legacySso;
+  } else if (parentPayerSpaces.length > 1) {
+    mediaProps.onClick = () => {
+      openModal('MULTI_PAYER_MODAL', {
+        title: 'Open Link as Payer',
+        name,
+        parentPayerSpaces,
+        link,
+        id,
+        spaceType: type,
+        metadata,
+      });
+    };
+  } else {
+    mediaProps.onClick = () => {
+      updateTopApps(id, type);
+      window.open(link.url[0] === '/' ? updateUrl(link.url, 'spaceId', spaceId) : link.url, link.target);
+    };
+  }
+
+  return [
+    {
+      ...space,
+      name,
+      description,
+      metadata,
+      link,
+    },
+    mediaProps,
+  ];
+};
+
+const Link = ({ spaceId, children, ...rest }) => {
+  const { loading } = useSpacesContext();
+  const [{ name, parents, metadata, description, activeDate } = {}, props = {}] = useLink(spaceId);
+  if(loading) return null;
+
+  const isNew = activeDate => dayjs().diff(activeDate,'day') < 30
+  return (
+    <Card title={name} className="mb-4 application" tabIndex={0} tag={CardLink} {...props} {...rest} aria-label={name}>
+      <CardBody className="card-block d-flex">
+        <span className="d-table-cell align-middle pr-2">
+          {/* <FavoriteHeart id={id} onChange={(_, e) => e.stopPropagation()} /> */}
```

Comment:
i think should be conditionally rendered if `props.allowFavorite` is true

Reply:
Yeah that is the end goal. We will have to note though, that you will need to have your `FavoritesProvider` somewhere up the tree in order to use this.